page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/PageManager.java

```diff
@@ -181,13 +145,13 @@ public static Page changeUrlByTitle(String title) throws PageInitializationExcep
      */
     public static Page changeUrlByTitle(String packageName, String title) throws PageInitializationException {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused method parameter "packageName". ([squid:S1172](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS1172))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=81&issues=AWU2kh_5W2AOP0CkD9zl&open=AWU2kh_5W2AOP0CkD9zl)

[ISSUE_KEY]: AWU2kh_5W2AOP0CkD9zl